Dockerfile

```diff
@@ -53,6 +53,27 @@ EXPOSE 8000
 ENTRYPOINT ["./docker-entrypoint.sh"]
 CMD ["python", "./cfgov/manage.py", "runserver", "0.0.0.0:8000"]
 
+# Build Frontend Assets
+FROM cfgov-dev as cfgov-build
+
+ENV STATIC_PATH ${APP_HOME}/cfgov/static/
+ENV PYTHONPATH ${APP_HOME}/cfgov
+
+# Django Settings
+ENV DJANGO_SETTINGS_MODULE cfgov.settings.production
+ENV DJANGO_STATIC_ROOT ${STATIC_PATH}
+ENV ALLOWED_HOSTS '["*"]'
+
+# See .dockerignore for details on which files are included
+COPY . .
+
+RUN yum -y install nodejs yarn && \
+    ./frontend.sh production && \
+    cfgov/manage.py collectstatic && \
+    yarn cache clean && \
+    rm -rf node_modules npm-packages-offline-cache
+
+
```

Comment:
This creates a container that installs node and yarn and builds the static assets, it also cleans up node_modules and npm-packages which can probably be deleted as we don't copy them into our production container.